HISTORY.md

```diff
@@ -4,6 +4,7 @@
 * Fix data corruption casued by output of intra-L0 compaction on ingested file not being placed in correct order in L0.
 
 ### Public API Change
+* Database generated will not be able to be opened by version 5.1 or older, even if no new feature is used.
```

Comment:
Is 5.1 the version where we introduced kNewFile4 manifest format?

Reply:
I thought it was 5.2, so I mentioned it here. But I rechecked and it seems to be introduced in 4.2: https://github.com/facebook/rocksdb/commit/b77eb16aba293728296378c6ae46a77d923ba24e

Then maybe I don't need to have this line. Let me double check.

Reply:
Yes I validated that release 4.2 can open the DB created by DB with this change. I'll remove this line.